Create Readme file

DavinciDelta opened the pull request:
Include a readme to give download instructions and features

DavinciDelta committed:
Create official readme

Include details of logging into application

DavinciDelta committed:
Update README.md

DavinciDelta committed:
Merge branch 'main' into readme

DavinciDelta committed:
Update Readme features

DavinciDelta committed:
Merge branch 'main' into readme

DavinciDelta committed:
Update viewing section

Update view count, opening and sending mail for the application

DavinciDelta committed:
Update table of content

Chandan8186 reviewed (changes requested):
Looks good overall, perhaps it would be good to also briefly mention the other 2 buttons on the index page and what they do. Also I think it will be good to add a disclaimer about following the user's email provider's own rate limit to prevent blacklisting e.g. 500 per day for gmail, yahoo

FireRadical22 committed:
Update README.md

Add screenshot of API permissions User must apply on their Microsoft Azure Application

Update README.md to include more details on what users should do in the event of unable to login via OAuth.

FireRadical22 committed:
Update README.md

Specify need for app password for gmail and yahoomail under 'logging into your account password' section

Add an additional note to remind users that not to log into their outlook account via this method.

FireRadical22 committed:
Update README.md

Add a note to inform users to use localhost url if logging in via Outlook

FireRadical22 committed:
Update README.md

Chandan8186 committed:
add google api scopes troubleshooting

Chandan8186 committed:
fix spelling error

Chandan8186 committed:
rephrase troubleshooting

DavinciDelta committed:
Include break lines

DavinciDelta committed:
Include video

bryanyee33 committed:
Update README

Chandan8186 committed:
amend placeholder error message

Chandan8186 committed:
amend source code instructions

Chandan8186 committed:
amend setup process

Chandan8186 committed:
fix TOC link

Chandan8186 committed:
amend readme title
